### PR TITLE
Fix package references in field type names

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**
This PR fixes package references in codegen for `(set_once)` option. Previously, the package of the message type where a field is declared was used instead of the package of the message class of the field type.

Also the PR bumps codegen module dependencies to fix the scope of the call of a referenced field.

### CODE REVIEW

File: java/src/main/kotlin/io/spine/validation/java/setonce/SetOnceJavaConstraints.kt [hunk 1]
```diff
@@ -82,7 +83,8 @@ internal sealed class SetOnceJavaConstraints(
     protected val fieldGetterName = "get$fieldNameCamel"
     protected val fieldSetterName = "set$fieldNameCamel"
     protected val fieldGetter = "$fieldGetterName()"
-    protected val declaringMessage = declaredIn.message.javaClassName(declaredIn.fileHeader)
+    protected val declaringMessage =
+        field.declaringType.toMessageType(typeSystem).javaClassName(typeSystem)
```
Comment (yevhenii-nadtochii) [inline java/src/main/kotlin/io/spine/validation/java/setonce/SetOnceJavaConstraints.kt:87]:
```suggestion
    protected val declaringMessage = field.declaringType
        .toMessageType(typeSystem)
        .javaClassName(typeSystem)
```

For me, it reads a bit easier when formatted this way.

File: java/src/main/kotlin/io/spine/validation/java/setonce/SetOnceMessageField.kt [hunk 2]
```diff
@@ -50,9 +51,9 @@ internal class SetOnceMessageField(
         }
     }
 
-    private val fieldTypeClass = field.type.message
-        .javaClassName(declaredIn.fileHeader)
-        .canonical
+    private val fieldTypeClass =
+        field.type.message.toMessageType(typeSystem).javaClassName(typeSystem)
+            .canonical
```
Comment (yevhenii-nadtochii) [inline java/src/main/kotlin/io/spine/validation/java/setonce/SetOnceMessageField.kt:56]:
```suggestion
    private val fieldTypeClass = field.type.message
            .toMessageType(typeSystem)
            .javaClassName(typeSystem)
            .canonical
```

The same suggestion.

**Review by yevhenii-nadtochii — APPROVED** [review]
Good catch.

I will cover external message fields as I add negative tests cases.

**Comment by alexander-yevsyukov** [comment]
It's the build of `core-java` who discovered the bug :)

**Pull request merged**